### PR TITLE
executor: fix issue #5132, update handle id with some other unique keys report Duplicate keys. (#5138)

### PR DESCRIPTION
**Pull request by winkyao**
cherry-pick

### CODE REVIEW

**Comment by winkyao** [comment]
@shenli @coocood PTAL

**Comment by winkyao** [comment]
@zimulala PTAL

File: table/tables/tables.go [hunk 1]
```diff
@@ -411,12 +411,12 @@ func (t *Table) genIndexKeyStr(colVals []types.Datum) (string, error) {
...
 
 // addIndices adds data into indices. If any key is duplicated, returns the original handle.
-func (t *Table) addIndices(ctx context.Context, recordID int64, r []types.Datum, bs *kv.BufferStore) (int64, error) {
+func (t *Table) addIndices(ctx context.Context, recordID int64, r []types.Datum, bs *kv.BufferStore, skipHandleCheck bool) (int64, error) {
 	txn := ctx.Txn()
 	// Clean up lazy check error environment
 	defer txn.DelOption(kv.PresumeKeyNotExistsError)
 	skipCheck := ctx.GetSessionVars().SkipConstraintCheck
-	if t.meta.PKIsHandle && !skipCheck {
+	if t.meta.PKIsHandle && !skipCheck && !skipHandleCheck {
```
Comment (zimulala) [inline table/tables/tables.go]: Can we use checkHandleExists directly?

Comment (winkyao) [reply table/tables/tables.go]: I will refactor it in another pr, not this.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**